build-release-tools/application/post-result.py

```diff
@@ -109,7 +107,7 @@ def generate_failure_report(build_url, build_name):
     
     return None
 
-def get_sub_builds(build_url, depth = 1):
+def get_sub_builds(build_url, FAIL_REPORTS, depth = 1):
```

Comment:
is there any reason you have to use the capital letters for the argument `FAIL_REPORTS`? If no, please change to lower case just like others.